Microsoft.Toolkit.Uwp.SampleApp/SamplePages/FrameworkElementEx/FrameworkElementExPage.xaml

```diff
@@ -0,0 +1,132 @@
+﻿<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.FrameworkElementExPage"
```

Comment:
This page doesn't really need to be fully filled out in full as long as you have some object with some reference to your extension properties.

Reply:
@michael-hawker fixed.